### PR TITLE
Update submiting snapshot endpoints

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
Intention of this PR is simplifying snapshot submitting procedure.
Updates:
- rename `/api/v0/admin/snapshot/{tag}` to `/api/v0/admin/snapshot/snapshot_info/{tag}`
- added new endpoint `/api/v0/admin/snapshot/raw_snapshot/{tag}` which submits and saves snapshot as previous endpoint does but from the `RawSnapshot` format.

So as a result if you have a `RawSnapshot` struct you can directly submit it into the vit service without transforming it to the `SnapshotInfo` struct with the usage of `catalyst-toolbox` tool.

### CODE REVIEW

**Comment by Mr-Leshiy** [comment]
Firstly want to merge this PR https://github.com/input-output-hk/vit-servicing-station/pull/259  and then update current

**Review by saibatizoku — APPROVED** [review]
Flow seems cleaner, and with better naming.

File: vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs [hunk 1]
```diff
@@ -115,7 +116,7 @@ mod test {
 
         assert_eq!(
             warp::test::request()
-                .path(format!("/snapshot/{}", "tag_a").as_ref())
+                .path(format!("/snapshot/snapshot_info/{}", "tag_a").as_ref())
```
Comment (cameron1024) [inline vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs:119]: Appreciate it's not your PR adding this, but could this just be `"/snapshot/snapshot_info/tag_a"`?

Comment (Mr-Leshiy) [reply vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs:119]: Oh, yea, will update it, no problem 😊

**Review by dkijania — CHANGES_REQUESTED** [review]
I'm missing an sanity test for importing RawSnapshot into vit-ss. Apart of that it's looking good

**Comment by Mr-Leshiy** [comment]
depends on https://github.com/input-output-hk/catalyst-toolbox/pull/163

**Pull request merged**